### ISSUE TITLE
Possibility to increase space key

### ISSUE THREAD

**Issue by gabriellluz** [AnySoftKeyboard/AnySoftKeyboard#1686]
### Steps to reproduce
1. Just type some text
2. Press space key

### Expected behaviour
Space key should be larger and having a comma and a dot in the same key, like SwiftKey or some other keyboards.

### Actual behaviour
The space key is so narrow, I end up pressing comma and dot keys.

**Android OS version:**
Android Pie - Pixel Experience ROM

**Device manufacturer and model:**
Moto G4 Play

**List of installed add-ons (like languages, or themes):**
Brazilian Portuguese language, German language, English and Spanish language. I use QWERTY layout.

**Comment by ubergeek77** [comment]
+1 For this. This seems like an amazing keyboard app, and I'm trying to stop using GBoard for privacy reasons, but typing on this layout is so cumbersome and difficult due to muscle memory. I had to switch to GBoard halfway through this comment just to type quickly :S

This issue seemed related, but it's over a year old, and the relevant option appears to have been removed:

https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1259

Somewhat related to this are two issues I have with this keyboard:

1. The emoji button doesn't pull up the list of emojis, it types a 😃 unless I long-press it. Is there a setting for this?

2. I want the special keys to be accessible via a button on the bottom-right, rather than a narrow button on the top.

Basically, I would like to skin this app to look more like GBoard. Is there a way to set a custom key layout? That would allow the user to fully customize the key layout to their liking.

Is there anything we can do to customize the keyboard layout, and what certain custom keys do? That would be fantastic.

---

Finally, only tangentially related to this, the minimum vibration strength is way too high. I would rather a "clicky" short vibration than a full vibration that this keyboard currently has. A slider for this would be better than just three options. I'm mentioning this since we're on the topic of keyboard customization, though that may be better suited for a separate issue.

**Comment by menny** [comment]
Thanks for the feedback. A couple of replies:

1) There are several options for the bottom row. Look under UI->Tweaks->Generic bottom row.
2) There is an option to flip between long-press and short-press outcome. Under Emoji Setup->Tweaks(top-right icon)->Long-press smiley popup
3) I'm planning on creating a slider to control vibration.

**Comment by ubergeek77** [comment]
Thank you for pointing that out! I totally missed the first point, I was looking in the wrong settings section.

Using the "simple" layout is definitely an improvement. However, I would like to customize the keys a bit more. I'd like the comma to be an emoji button, and the comma to be accessible from a long-press on the period button.

Rather than have you make special keyboards for everyone, is there a good way to create/preview custom, user-created layouts and import them? I wouldn't mind spending a lot of time tweaking my own custom layout with key width/height key placement, long press menu, and etc.

**Comment by cr1za** [comment]
> Using the "simple" layout is definitely an improvement. However, I would like to customize the keys a bit more. I'd like the comma to be an emoji button, and the comma to be accessible from a long-press on the period button.
> 
> Rather than have you make special keyboards for everyone, is there a good way to create/preview custom, user-created layouts and import them? I wouldn't mind spending a lot of time tweaking my own custom layout with key width/height key placement, long press menu, and etc.

Please, we need an answer for this. Even with the simple layout, there could be more settings to be able to change the bottom row more specifically. As well the settings are a mess, and I as an normal user do not understand or find half of the options....

**Comment by Joshuasony** [comment]
Basically, we want the Gboard layout, but for anysoftkeyboard and our language pack. Does somebody know a way how to achieve this? Similarity would already be an improvement. But I guess I have to open another issue.
![gboard](https://user-images.githubusercontent.com/7139459/99877008-55ba7200-2bfb-11eb-8985-4b58d4b97b80.png)
